internal/pkg/slicer/slicer.go

```diff
@@ -126,8 +121,8 @@ func SliceTable(logger log.Logger, table Table) (err error) {
 		return fmt.Errorf("error when reading CSV \"%s\": %w", table.InPath, err)
 	}
 
-	// Write manifest
-	if err := manifest.WriteTo(table.OutManifestPath); err != nil {
+	// Close the writer
+	if err = writer.Close(); err != nil {
```

Comment:
Teraz sa to deje v spravnom poradi:
- `Close the writer`: ukonci sa writer, teda vsetko je zapisane
- Spocita sa velkost (gzip) suborov v dir, este tam nie je manifest.
- Zapise sa manifest.
- Zaloguje sa msg.


![image](https://github.com/keboola/processor-split-table/assets/19371734/709fe3f1-bb9b-41a5-8f01-8ab600f4b0ef)